QUEST_JOBSTEP.tsv

```diff
@@ -3013,118 +3013,118 @@ QUEST_JOBSTEP_20150729_003012
-QUEST_JOBSTEP_20150803_003114	You've heard about eternal lives from Amjinases. Go back to Sorcerer Master.
-QUEST_JOBSTEP_20150803_003115	Obtain the keepstakes which you could feel strong regrets
-QUEST_JOBSTEP_20150803_003116	Necromancer Master wants to have the keepstake with stronger grudge. Obtain the keepstakes that you can feel strong regrets from Bidentis Shrine.
-QUEST_JOBSTEP_20150803_003117	You've collected the keepstakes with strong regrets. Go back to Necromancer Master.
-QUEST_JOBSTEP_20150803_003118	Go meet Bidotas in Siaulai Mine Village.
-QUEST_JOBSTEP_20150803_003119	Bidotas told you that in order to investigate Crystal Mine, he needs the orb that records the memories of the past. Obtain the orb that records the memories of the past from Chronomancer Master.
-QUEST_JOBSTEP_20150803_003120	You've borrowed the orb that records the memories of the past. Go back to Bidotas.
-QUEST_JOBSTEP_20150803_003121	You've obtained the orb that records the memories of the past. Talk with Bidotas.
-QUEST_JOBSTEP_20150803_003122	Bidotas wants you to investigate the incident that occured in Crystal Mine. With the glass orb that reflects the past which Chronomancer Master gave to you, accomplish the exploration rate 100% in all regions of the 1st and 2nd district of Crystal Mine.
-QUEST_JOBSTEP_20150803_003123	The memories of Crystal Mine are recorded in the orb. Go back to Bidotas.
-QUEST_JOBSTEP_20150803_003124	Give the present secretly to Scout Master
-QUEST_JOBSTEP_20150803_003125	Rogue Master wants to give a surprise present to Scout Master who once was very close to him. Go find Scout Master in Gateway of Great King and put the present secretly in his pocket.
-QUEST_JOBSTEP_20150803_003126	You've succeeded putting the present secretly without being detected. Go back to Rogue Master.
-QUEST_JOBSTEP_20150803_003127	Search for the traces of Lydia Schaffen
-QUEST_JOBSTEP_20150803_003128	Fletcher Master wants to look for the bow which the first Fletcher Master made for Lydia Schaffen. Find the bow of the first Flectcher Master from 31_1 of Canyon Area.
-QUEST_JOBSTEP_20150803_003129	Hand it over to Fletcher Master
-QUEST_JOBSTEP_20150803_003130	You've found the bow of the first Fletcher master. Go back to Fletcher Master.
+QUEST_JOBSTEP_20150803_003113	The Sorcerer Master wants to hear more from Amjinases. You can meet the Dievdirbys Master in the west forest of Siaulai, the Necromancer Master in Mulus Corridor, and the Chronomancer Master in Stele Road.
+QUEST_JOBSTEP_20150803_003114	You've heard about eternal lives from Amjinases. Return to the Sorcerer Master.
+QUEST_JOBSTEP_20150803_003115	Obtain the keepstakes which you could feel strong regrets from.
+QUEST_JOBSTEP_20150803_003116	The Necromancer Master wants to have the keepstake with stronger grudge. Obtain the keepstakes that you can feel strong regrets from Bidentis Shrine.
```

Comment:
I believe these are "keepsakes".
Maybe this would make more sense if it were changed to:
"The Necromancer Master wants to collect the keepsakes with strong grudges within them.  Obtain the keepsakes that you can feel strong regrets within at Bidentis Shrine."